### PR TITLE
Bump to version 2.0.0

### PR DESCRIPTION
**Pull request by JinnyYi**

### CODE REVIEW

**Comment by Xuanwo** [comment]
Please merge https://github.com/beyondstorage/go-service-s3/pull/95 into this branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
...
+- *: Implement GSP-47 Additional Error Specification (#78)
+- *: Implement GSP-51 Distinguish Errors by IsInternalError (#83)
+- storage: Implement GSP-61 Add object mode check for operations (#89)
+
+### Changed
+
+- build(deps): bump github.com/aws/aws-sdk-go from 1.38.27 to 1.38.45 (#73)
+- storage: Idempotent storager delete operation (#82)
+- storage: Implement GSP-62 WriteMultipart returns Part (#88)
+- *: Implement GSP-73 Organization rename (#94)
```
Comment (Xuanwo) [inline CHANGELOG.md:21]: Please add SDK upgrade into the changelog.

Comment (JinnyYi) [reply CHANGELOG.md:21]: `aws-sdk-go` upgraded severval times, do I need to list out all the updates?

Comment (Xuanwo) [reply CHANGELOG.md:21]: Only add the latest version is enough, like:

```
- Bump aws-go-sdk to 1.38.45
```

**Pull request merged**